examples/pubsub-demo/build.gradle

```diff
@@ -1,20 +1,13 @@
 plugins {
-    id 'application'
     id 'groovy'
-    id 'org.grails.grails-web'
-    id 'org.grails.plugins.views-json'
 }
 
+version = rootProject.version
 group = 'pubsub.demo'
-configurations.all {
-
-    resolutionStrategy.dependencySubstitution {
-        substitute module("org.codehaus.groovy:groovy") using module('org.apache.groovy:groovy:4.0.22')
-        substitute module("org.codehaus.groovy:groovy-templates") using module('org.apache.groovy:groovy-templates:4.0.22')
-        substitute module("org.codehaus.groovy:groovy-xml") using module('org.apache.groovy:groovy-xml:4.0.22')
-        substitute module("org.codehaus.groovy:groovy-json") using module('org.apache.groovy:groovy-json:4.0.22')
-    }
-}
+
+// Must be applied after version is set
```

Comment:
@matrei I think this is only for the publish plugin.  For the gsp / web plugins this isn't a requirement.

Reply:
Yes, you are right, I mixed it up
#171